MAINTAINERS

```diff
@@ -240,6 +256,11 @@ example and you'll be fine".
 		"stevvooe",
 	]
 
+	[Org.Projects]
```

Comment:
Maybe we could skip this part so we don't have to keep a running list, i know this is something we will forget to update.